### PR TITLE
Adds optionals section on README for step 3-networks

### PR DESCRIPTION
**Pull request by amandakarina**
Hey Folks

This PR highlights the toggles for optional services on step 3-network.

Can you take a look? 

Thanks!

### CODE REVIEW

File: 3-networks/README.md [hunk 1]
```diff
@@ -128,3 +128,14 @@ and only the corresponding environment is applied.
 1. Run `./tf-wrapper.sh apply development`
 
 If you received any errors or made any changes to the Terraform config or any `.tfvars`you must re-run `./tf-wrapper.sh plan <env>` before run `./tf-wrapper.sh apply <env>`
+
+### Optionals
+
+This module provides some optional network configurations. You can check below what are they and what will be created if you enable each one:
+
+1. **nat_enabled:** by default, this option is `false`. If you change to `true`, it will create: Compute Cloud router, Compute Address, Compute Router NAT and tag based static route through IGW to access internet. For this, you have to input also `nat_bgp_asn, nat_num_addresses_region1, nat_num_addresses_region2`.
```
Comment (atos-cit) [inline 3-networks/README.md:136]: NIT: I would use a unordered list

Comment (amandakarina) [reply 3-networks/README.md]: Agree!

**Comment by rjerrems** [comment]
Just for clarity, this is also pending the same feedback mentioned here https://github.com/terraform-google-modules/terraform-example-foundation/pull/239

File: 3-networks/README.md [hunk 2]
```diff
@@ -128,3 +128,7 @@ and only the corresponding environment is applied.
 1. Run `./tf-wrapper.sh apply development`
 
 If you received any errors or made any changes to the Terraform config or any `.tfvars`you must re-run `./tf-wrapper.sh plan <env>` before run `./tf-wrapper.sh apply <env>`
+
+### Optional Variables
```
Comment (rjerrems) [inline 3-networks/README.md:132]: If @atos-cit agrees with my feedback here https://github.com/terraform-google-modules/terraform-example-foundation/pull/239, I think we can remove this.

**Review by rjerrems — APPROVED** [review]
One small nit, but otherwise LGTM

File: 3-networks/README.md (hunk 2 above)
Comment (atos-cit) [reply 3-networks/README.md]: Removed the section as agreed in #239 .

**Pull request merged**